simtools/data_model/metadata_collector.py

```diff
@@ -35,7 +38,7 @@ class MetadataCollector:
 
     """
 
-    def __init__(self, args_dict, data_model_name=None):
+    def __init__(self, args_dict, metadata_file_name=None, data_model_name=None):
```

Comment:
This covers the case where the meatdata_collector is not called from an application with all in the required information in `args_dict`. Use `metadata_file_name`, we can simple read a file and extract the metadata.

Reply:
Ok, thanks.